README.md

```diff
@@ -1,36 +1,212 @@
 # heat-operator
-The Heat Operator deploys the OpenStack Heat project in a Kubernetes cluster.
+
+The Heat Operator deploys the OpenStack Heat project in a OpenShift cluster.
 
 ## Description
-This project should be used to deploy the OpenStack Heat project. It expects that there is an existing Database and Keystone service available to connect to. The config can be tuned using the `defaultConfigOverwrite` interface to override default config option in the `heat.conf` file.
 
+This project should be used to deploy the OpenStack Heat project. It expects that there is an existing Database and Keystone service available to connect to. The config can be tuned using the `defaultConfigOverwrite` interface to override default config option in the `heat.conf` file.
 
 ## Getting Started
-You’ll need a Kubernetes cluster to run against. You can use [KIND](https://sigs.k8s.io/kind) to get a local cluster for testing, or run against a remote cluster.
-**Note:** Your controller will automatically use the current context in your kubeconfig file (i.e. whatever cluster `kubectl cluster-info` shows).
+
+This operator is deployed via Operator Lifecycle Manager as part of the OpenStack Operator bundle:
+https://github.com/openstack-k8s-operators/openstack-operator
+
+To configure Heat specifically, we expose options to add custom configuration and the number of replicas for both Heat API and Heat Engine:
+
+The following is taken from the Sample config in this repo:
+
+```yaml
+spec:
+  customServiceConfig: "# add your customization here"
+  databaseInstance: openstack
+  databaseUser: "heat"
+  rabbitMqClusterName: rabbitmq
+  debug:
+    bootstrap: false
+    dbSync:
+    service: false
+  heatAPI:
+    containerImage: "quay.io/podified-antelope-centos9/openstack-heat-api:current-podified"
+    customServiceConfig: "# add your customization here"
+    databaseUser: "heat"
+    debug:
+      bootstrap: false
+      dbSync: false
+      service: false
+    passwordSelectors:
+      service: AdminPassword
+      database: AdminPassword
+    replicas: 1
+    resources: {}
+    secret: "osp-secret"
+    serviceUser: ""
+  heatEngine:
+    containerImage: "quay.io/podified-antelope-centos9/openstack-heat-engine:current-podified"
+    customServiceConfig: "# add your customization here"
+    databaseUser: "heat"
+    debug:
+      bootstrap: false
+      dbSync: false
+      service: false
+    passwordSelectors:
+      service: AdminPassword
+      database: AdminPassword
+    replicas: 1
+    resources: {}
+    secret: "osp-secret"
+    serviceUser: ""
+  passwordSelectors:
+    service: AdminPassword
+    database: AdminPassword
+  preserveJobs: false
+  secret: osp-secret
+  serviceUser: "heat"
+```
+
+If we want to make some changes to the Heat configuration, for example, changing the number of `heat_engine_workers`.
```

Comment:
I'm asking this mostly from my curiosity but where this `heat_engine_workers` term come from ?
If this is not pre-defined somewhere then we probably use more generic term such as `heat-engine workers` instead.